src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
```

Comment:
if you don't have anythig inside <div>, you need to write it as self closed component (<div class="header__logo" />). This rule is about all components, that are in need with closing tag.